app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,23 @@
-<h2>Log in</h2>
+<div class="flex flex-col items-center sm:card py-20 px-8 w-full sm:w-auto ">
+  <div class="flex flex-col items-center gap-12">
+    <h2 class="text-gray-3">Welcome back!</h2>
+    <p class="text-gray-3 font-normal">Don't have an account?
+      <%= link_to 'Sign Up', new_registration_path(resource_name) %>
+    </p>
+    <%= form_for(resource, html: { class: 'user pt-6'} , as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
`form_for` is softly deprecated, use `form_with` instead. Also, mind the resource you're using. What is `resource` and `resource_name`?

Reply:
`resource` and `resource_name` are the devise helpers that just points to the User model